version/version.go

```diff
@@ -0,0 +1,5 @@
+package version
+
+var (
+	Version = "0.0.1"
```

Comment:
I think this is intended to be our own product version. And maybe it should be printed in the printVersion function in main.go.

So we should change this to be 0.3.0 for now. And update the kd-release-process.md document to include the necessity of updating this file.

We could think about ways to get cleverer with this but I think that's fine for now.

Reply:
Done.